clang/lib/Parse/ParseExprCXX.cpp

```diff
@@ -3639,6 +3639,10 @@ ExprResult Parser::ParseRequiresExpression() {
                 //  TryParseParameterDeclarationClause).
```

Comment:
I don't think the above fixme above is needed, I don't think there is any other way to handle this, at least based on some of the other places `TryParseParameterDeclarationClause()` is used e.g. `TryParseFunctionDeclarator(...)`